t/08-autotest.t

```diff
@@ -334,6 +334,20 @@ subtest python => sub {
     } qr/Traceback.*No module named.*thismoduleshouldnotexist.*/s, 'Python traceback logged';
 };
 
+subtest 'python run_args' => sub {
+    %autotest::tests = ();
+    my $targs = OpenQA::Test::RunArgs->new();
+    $targs->{data} = 23;
+
+    autotest::loadtest('tests/pythontest_with_runargs.py', run_args => $targs);
+
+    loadtest 'pythontest_with_runargs.py';
+    my $p1 = $autotest::tests{'tests-pythontest_with_runargs'};
+
+    stderr_like { $p1->runtest } qr{run_args are not supported in python}, 'Expected output from pythontest_with_runargs.py';
```

Comment:
This is checking that the test test prints the message. I don't see how that tells us that all tests using run_args will fail cleanly.

Reply:
I would expect that the test aborts because run_args was set before even executing the test code.

Reply:
It does, https://github.com/os-autoinst/os-autoinst/blob/master/basetest.pm#L349 calls `run` with parameters. If the python function does not handle them it should raise an error on the python side.

I can add a test for that too.

Reply:
ugh, I just saw my mistake :facepalm:

Reply:
Is there a way to mark a test module as a python test? Or assert that? I can't find anything like that, all tests look pretty equal 🤔

Reply:
@josegomezr https://github.com/os-autoinst/os-autoinst/blob/master/autotest.pm#L148 seems explicitly aware of calling Python code already?

Reply:
@kalikiana Thanks for the pointer!

Check 34bb021, it's using now `$is_python` to bail if `run_args` is provided.

And that way `def run(self):` is completely stable :D

Reply:
Excellent! Thank you for making the changes. Looks awesome!